src/FloorSubsystem/FloorTest.java

```diff
@@ -0,0 +1,32 @@
+package FloorSubsystem;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+class FloorTest {
+
```

Comment:
look into the setup() and teardown() methods, they are usually used for JUnit

Reply:
Don't think this is necessary as of yet but will implement if test grow larger and all require same instantiations.